user-impl/src/main/java/com/example/auction/user/impl/UserServiceImpl.java

```diff
@@ -1,18 +1,26 @@
 package com.example.auction.user.impl;
 
 import akka.NotUsed;
+import akka.japi.Pair;
+import akka.stream.javadsl.Source;
 import com.example.auction.pagination.PaginatedSequence;
 import com.example.auction.user.api.User;
+import com.example.auction.user.api.UserEvent;
 import com.example.auction.user.api.UserRegistration;
 import com.example.auction.user.api.UserService;
 import com.lightbend.lagom.javadsl.api.ServiceCall;
+import com.lightbend.lagom.javadsl.api.broker.Topic;
+import com.lightbend.lagom.javadsl.broker.TopicProducer;
+import com.lightbend.lagom.javadsl.persistence.AggregateEventTag;
+import com.lightbend.lagom.javadsl.persistence.Offset;
 import com.lightbend.lagom.javadsl.persistence.PersistentEntityRef;
 import com.lightbend.lagom.javadsl.persistence.PersistentEntityRegistry;
 
 import javax.inject.Inject;
 import java.time.Instant;
 import java.util.Optional;
 import java.util.UUID;
+import java.util.concurrent.CompletableFuture;
 
```

Comment:
I think there's some unused imports here. I understand in previous commits those were used but after removing the Kafka client from the web-gateway they may be not necessary anymore.